rust_src/src/frames.rs

```diff
@@ -727,6 +727,33 @@ pub fn make_frame_invisible(frame: LispFrameLiveOrSelected, force: bool) {
     }
 }
 
+#[lisp_fn(min = "0")]
```

Comment:
No need for the min=, `lisp_fn` handles checking the number of arguments. Use `min` when a function can take less parameters than defined like say this function:
`pub fn takes_optional_arguments(first: LispObject, second: LispObject, third: LispObject)`

That should fail if not given 3 parameters. But `min=1` would let `second` and `third` be omitted and they will be sent as `Qnil`.

Reply:
I see. Thanks ;)